docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
```

Comment:
As far as I remember, getting GPUs to be accessible through docker requires a dedicated binary? Or is that just a FAIR thing?

Reply:
Well, this image will run with both `nvidia-docker` and `docker` (e.g. even on MacOS), but I believe recent versions of `docker` have a `--gpus` flag by default (which otherwise requires `nvidia-docker`).